docs/source/quicktour.mdx

```diff
@@ -39,40 +54,47 @@ tokenizer_name_or_path = "bigscience/mt0-large"
 model = AutoModelForSeq2SeqLM.from_pretrained(model_name_or_path)
 ```
 
-3. Preprocess your model if you use [`bitsandbytes`](https://github.com/TimDettmers/bitsandbytes) for `int8` quantized training; otherwise, skip this step. 
+<Tip>
```

Comment:
Do we need this tip in the quick tour?

Reply:
I don't think it's an essential feature of the library so I think we can remove this. I am also beginning to think the same thing about the section with the DeepSpeed integration, and I made a separate guide for it in #355.

Let's wait and see what @pacman100 thinks :)

Reply:
Hello Maria and Steven, I think we can remove this as this is explained in the INT8-related task guide. Same goes for the DeepSpeed integration part as that too will be covered in separate guide